CactbotOverlay/VersionChecker.cs

```diff
@@ -12,6 +12,7 @@ namespace Cactbot {
   class VersionChecker {
     private ILogger logger_ = null;
 
+    public const string kReleaseAPIEndpointUrl = @"https://api.github.com/repos/quisquous/cactbot/releases/latest";
```

Comment:
This is the tiniest of style nits, but this variables has two acronyms, but one is all caps and the other not.  Maybe kReleaseApiEndpointUrl? <_<